client/my-sites/purchases/main.tsx

```diff
@@ -50,6 +54,10 @@ export function Purchases() {
 	const translate = useTranslate();
 	const siteSlug = useSelector( getSelectedSiteSlug );
 	const logPurchasesError = useLogPurchasesError( 'site level purchases load error' );
+	const nextAppointment = useSelector( getConciergeNextAppointment );
+	const isUserBlocked = useSelector( getConciergeUserBlocked );
+	const availableSessions = useSelector( getAvailableConciergeSessions );
```

Comment:
Naming nits: it's not clear from the ~variable~ const names what the items refer to. Given that we're on a payment-related page, `isUserBlocked` is almost certainly not specific enough.

I'd vote for more specific names for all three, but especially for `isUserBlocked`.
```suggestion
	const nextConciergeAppointment = useSelector( getConciergeNextAppointment );
	const isConciergeUserBlocked = useSelector( getConciergeUserBlocked );
	const availableConciergeSessions = useSelector( getAvailableConciergeSessions );
```